qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -402,6 +392,14 @@ def properties(
             api_properties = self._api_client.backend_properties(self.name, datetime=datetime)
             if not api_properties:
                 return None
+            if "gates" in api_properties and isinstance(api_properties["gates"], list):
+                if (
+                    self.options.use_fractional_gates is not None
+                    and not self.options.use_fractional_gates
+                ):
+                    api_properties["gates"] = [
+                        g for g in api_properties["gates"] if g.get("gate") not in FRACTIONAL_GATES
+                    ]
```

Comment:
You address properties here, and not inside `properties_from_server_data`. But, for the configuration, you did it inside `configuration_from_server_data`. Is there a reason? The code will be more readable if the structure is uniform.

Reply:
Good point. I shifted the filtering into `properties_from_server_data` for consistency.